core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -452,14 +455,22 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
         producer.flush()
         assertTrue("All request are complete.", responses.forall(_.isDone()))
         // Check the messages received by broker.
-        val fetchResponse = if (leader.get == configs.head.brokerId) {
-          consumer1.fetch(new FetchRequestBuilder().addFetch(topic, 0, 0, Int.MaxValue).build())
-        } else {
-          consumer2.fetch(new FetchRequestBuilder().addFetch(topic, 0, 0, Int.MaxValue).build())
+        val records = new ArrayList[ConsumerRecord[Array[Byte], Array[Byte]]]()
+        val maxIters = 50
+        var iters = 0
+        consumer1.seek(new TopicPartition(topic, 0),0)
```

Comment:
Space missing after `,`. Do we need the `seek` here given that we are seeing to offset 0?